easybuild/tools/options.py

```diff
@@ -235,6 +236,7 @@ def config_options(self):
             'avail-repositories': ("Show all repository types (incl. non-usable)",
                                    None, "store_true", False,),
             'buildpath': ("Temporary build path", None, 'store', mk_full_default_path('buildpath')),
+            'external-modules-metadata': ("File specifying metadata for external modules", None, 'store', None),
```

Comment:
use a comma spearated list of files, also specify the format (eg `INI File(s) ...`)